docs/proposals/quic-design.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Quic Design
+status: Alpha
+authors:
+    - "@Qianchenjie"
+approvers:
+creation-date: 2019-05-01
+last-updated: 2019-05-08
+---
+
+# Enhanced Cloud/Edge Communication
+
+## Abstract
+In order to enhance cloud and edge communication efficiency, we introduced the [QUIC](https://quicwg.org/ops-drafts/draft-ietf-quic-applicability.html) protocol for cloudhub and edgehub.
+It is suggested that cloudHub should support both websocket and QUIC protocol access at the same time. And the edgehub can choose one of the protocols to access to the cloudhub.
+QUIC is a new transport which reduces latency compared to that of TCP. On the surface, QUIC is very similar to TCP+[TLS](http://technet.microsoft.com/en-us/library/cc785811.aspx)+HTTP/2 implemented on [UDP](http://c3lab.poliba.it/images/3/3b/QUIC_SAC15.pdf). Because TCP is implemented in operating system kernels, and middlebox firmware,making significant changes to TCP is next to impossible.However,since QUIC is built on top of UDP,it suffers from no such limitations.
+Key features of QUIC:
+
+1. [Dramatically reduced connection establishment time](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7867726)
+2. Improved congestion control
+3. [Multiplexing without head of line blocking](https://docs.google.com/document/d/1RNHkx_VvKWyWg6Lr8SZ-saqsQx7rFV-ev2jRFUoVD34/mobilebasic?pli=1)
+4. Forward error correction
+5. Connection migiation
+
+<img src="../images/proposals/quic-tcp-udp.png">
+<img src="../images/proposals/quic-connection.png">
+
+## Motivation
+For the edge side, the node network signal instability, TCP + TLS overhead is very large, QUIC'S 0-RTT can greatly increase access speed. 
+Due to the characteristics of multiplexing, business processing capacity can be greatly improved.
+
+## Use cases
+<img src="../images/proposals/quic-design.png">
+
+### Start the websocket server
+1. User edit controller.yaml, modify cloudhub.protocol_websocket: true, and port: 10000.
+2. Running the cloudhub, and start the websocket server.
+3. Waiting for client access through websocket protocol.
+
+### Start the quic server
+1. User edit controller.yaml, modify cloudhub.protocol_quic: true, and quic_port: 10001
+2. Running the cloudhub, and start the quic server.
+3. Waiting for client access through quic protocol.
+
+### Start the websocket and quic server
+1. User edit controller.yaml, modify cloudhub.protocol_quic: true, and quic_port: 10001， cloudhub.protocol_websocket: true, and port: 10000
```

Comment:
see previous comments for websocket server step.

Reply:
done ， Start the websocket and quic server at the same time